packages/openassistants/openassistants/llm_function_calling/entity_resolution.py

```diff
@@ -35,8 +35,8 @@ def entity_to_document(entity: IEntity) -> Document:
         metadata=dict(id=entity.get_identity()), page_content=entity.get_identity()
     )
 
-    if entity.get_description:
-        doc.page_content += f" ({entity.get_description})"
+    if entity.get_description():
```

Comment:
Good case for using the Walrus operator https://realpython.com/python-walrus-operator/